src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,5329 @@
+                <div class="chi-data-table__cell -name -bold">
+                  <div>Name 3</div>
+                </div>
+                <div class="chi-data-table__cell -name">
+                  <div>name-3</div>
+                </div>
+                <div class="chi-data-table__cell">
+                  <div>Nov 5, 2020 10:15 AM</div>
+                </div>
+              </div>
+            </div>
+            <div class="chi-data-table__footer">
+              <nav class="chi-pagination -compact" role="navigation" aria-label="Pagination">
+                <div class="chi-pagination__content">
+                  <div class="chi-pagination__start">
+                    <div class="chi-pagination__results">
+                      <span class="chi-pagination__label">240 Results</span>
+                    </div>
+                  </div>
+                  <div class="chi-pagination__center">
+                    <div class="chi-pagination__button-group chi-button-group">
+                      <a class="chi-button -icon -flat -sm" href="#" aria-label="Previous page">
+                        <div class="chi-button__content">
+                          <i class="chi-icon icon-chevron-left" aria-hidden="true"></i>
+                        </div>
+                      </a>
+                    </div>
+                    <div class="chi-pagination__label">
+                      <strong>2</strong>
+                      <span>of</span>
+                      <strong>3</strong>
+                    </div>
+                    <div class="chi-pagination__button-group chi-button-group">
+                      <a class="chi-button -icon -flat -sm" href="#" aria-label="Next page">
+                        <div class="chi-button__content">
+                          <i class="chi-icon icon-chevron-right" aria-hidden="true"></i>
+                        </div>
+                      </a>
+                    </div>
+                  </div>
+                  <div class="chi-pagination__end"></div>
+                </div>
+              </nav>
+            </div>
+          </div>
+        </div>
+      </div>
+
+h3 Portal -- Accordion
+.example.-mb--3
+  .-p--3
+    .chi-card.-portal
+      .chi-card__header.-sm.-pl--9
+        .chi-card__title Label
+      .chi-card__content.-p--0
+        .chi-data-table.-portal.-compact
+          .chi-data-table__head
+            .chi-data-table__row
+              .chi-data-table__cell.-expandable
+              .chi-data-table__cell
+                span Name
+              .chi-data-table__cell
+                span ID
+              .chi-data-table__cell
+                span Last Login
+          .chi-data-table__body
+            .chi-data-table__row.-collapsed
+              .chi-data-table__cell.-expandable
+                button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+                  .chi-button__content
+                    i.chi-icon.icon-plus.-xs
```

Comment:
I think `-sm` on `chi-button` sizes the icon to the equivalent of `-xs` by default. We should be able to remove all instances of this class as long as `-sm` is present on the button.
```suggestion
                    i.chi-icon.icon-plus
```